src/planwise/component/projects.clj

```diff
@@ -12,11 +14,31 @@
 (defn get-db [service]
   (get-in service [:db :spec]))
 
+;; Mapper functions
+
+(defn db->project
+  "Transforms a record retrieved from the database to make it usable from inside
+  de application"
+  [record]
+  (-> record
+      (set/rename-keys {:region_id :region-id
+                        :region_name :region-name})
+      (update :stats edn/read-string)
+      (update :filters edn/read-string)))
```

Comment:
Not sure I like edn as a format for storing data in psql, having hstore supported natively; though this works, and we'll seldom want to query by any of these fields.